openmdao/core/system.py

```diff
@@ -282,8 +283,8 @@ def pre_setup(self, problem):
         User-configurable method to be run when problem.setup() is called
         but prior to any actual problem setup.
 
-        Args
-        ----
+        Parameters
+        ----------
```

Comment:
Both `Args` and `Parameters` will WORK in the auto-doc, but I did the monkeypatch to add `Args` to numpydoc to avoid the confusion between `Params` and `params`